django/views/debug.py

```diff
@@ -158,6 +160,14 @@ def get_safe_request_meta(self, request):
             return {}
         return {k: self.cleanse_setting(k, v) for k, v in request.META.items()}
 
+    def get_safe_cookies(self, request):
```

Comment:
I wonder if it's worth documenting more of the available methods here? (Only *some* are currently listed, but I'm not sure why we stopped exactly there.) 

(Not essential for this PR. More a 🤔)